index.html

```diff
@@ -8,10 +9,10 @@ <h1>Introduction</h1>
     <p>
         "Where can I learn how to use pygame?" - one of the 
         most commonly asked questions that I get. But I always struggle to 
-        answer, I sometimes point them to the docs and examples, but that 
+        answer. I sometimes point them to the docs and examples, but that 
         isn't a comprehensive guide on building good pygame based applications.
         There are many other tutorials and guides I found, but I always 
-        find something lacking, there are many comments we give and many nitpicks we have 
+        find something lacking.  There are many comments we give and many nitpicks we have 
```

Comment:
Past tense and present tense used together, should _found_ be changed to _find_ or _have found_ in which case the other _find_ changes to _found_? or _..., but there was always something lacking._ ?

Reply:
Maybe this fix should be included with the first task in #63 since that's going to be editing the same content

Reply:
It might just be me, but I don't see anything wrong with the tense.

Reply:
Swapping between "I" and "we"

Reply:
Alright, maybe just me then. ChatGPT said it couldn't find any issues either, so I guess this is resolved then and stays as is